posthog/models/funnel.py

```diff
@@ -112,7 +112,8 @@ def _build_query(self, query_bodies: dict):
         ON_TRUE = "ON TRUE"
         LEFT_JOIN_LATERAL = "LEFT JOIN LATERAL"
         QUERY_HEADER = "SELECT {people}, {fields} FROM "
-        LAT_JOIN_BODY = """({query}) {step} {on_true} {join}"""
+        LAT_JOIN_BODY = """({query}) {step} {on_true} {join}""" if len(query_bodies) > 1 else """({query}) {step} {on_true} """
+        print(LAT_JOIN_BODY)
```

Comment:
probably want to nuke the print

Reply:
haha yeah i just realized there was a bunch of associated changes in this. gotta nuke the package lock too